gpytorch/likelihoods/gaussian_likelihood.py

```diff
@@ -38,7 +38,7 @@ def _shaped_noise_covar(self, base_shape: torch.Size, *params: Any, **kwargs: An
 
     def forward(self, function_samples: Tensor, *params: Any, **kwargs: Any) -> base_distributions.Normal:
         observation_noise = self._shaped_noise_covar(function_samples.shape, *params, **kwargs).diag()
-        return base_distributions.Normal(function_samples, observation_noise)
+        return base_distributions.Normal(function_samples, observation_noise.sqrt())
```

Comment:
Fortunately, `forward` for Gaussian likelihood is not used anywhere in the library because `marginal` has an analytical form hence why this survived :-).